src/components/project/DetailButton/index.js

```diff
@@ -5,17 +5,18 @@ import CloudinaryImage from '../../CloudinaryImage';
 
 import s from './styles.module.scss';
 
-const ProjectDetailButton = ({ text, className, href, ...rest }) => (
-  <a className={classnames(s.btn, className)} href={href} {...rest}>
+const ProjectDetailButton = ({ text, className, href, child = false, ...rest }) => (
+  <a className={classnames(child ? s.btnChild : s.btn, className)} href={href} {...rest}>
     {text}
-    <CloudinaryImage filename="github.svg" filters={[]} className={s.github} alt="GitHub" />
-    <CloudinaryImage filename="arrow.svg" filters={[]} className={s.arrow} alt={text} />
+    <CloudinaryImage filename="github.svg" filters={[]} className={child ? s.github : s.githubChild} alt="GitHub" />
+    <CloudinaryImage filename="arrow.svg" filters={[]} className={child ? s.arrow: s.arrowChild} alt={text} />
```

Comment:
Wouldn't make sense to use `classnames` instead of a ternary? feels cleaner IMO

Reply:
added @crissoca